isolate_example/lib/page_one.dart

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019-present the Flutter authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:flutter/material.dart';
+
+class PerformancePage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Column(
+        mainAxisSize: MainAxisSize.min,
+        children: <Widget>[
+          SmoothAnimationWidget(),
+        ],
+      ),
+    );
+  }
+}
+
+class SmoothAnimationWidget extends StatefulWidget {
+  @override
+  SmoothAnimationWidgetState createState() => SmoothAnimationWidgetState();
+}
+
+class SmoothAnimationWidgetState extends State<SmoothAnimationWidget>
+    with TickerProviderStateMixin {
+  AnimationController _controller;
+  var borderRadius;
+
+  @override
+  void initState() {
+    super.initState();
+
+    _controller =
+        AnimationController(duration: const Duration(seconds: 1), vsync: this)
+          ..addStatusListener(
+            (status) {
+              if (status == AnimationStatus.completed) {
+                _controller.reverse();
+              } else if (status == AnimationStatus.dismissed) {
+                _controller.forward();
+              }
+            },
+          );
+
+    borderRadius = BorderRadiusTween(
+      begin: BorderRadius.circular(100.0),
+      end: BorderRadius.circular(0.0),
+    ).animate(
+      CurvedAnimation(
+        parent: _controller,
+        curve: Curves.linear,
+      ),
+    );
+
+    _controller.forward();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return AnimatedBuilder(
+      animation: borderRadius,
+      builder: (BuildContext context, Widget child) {
```

Comment:
In general, we don't type the parameters for anonymous functions like this one. `builder: (context, child) {}` should be enough for the reader.